lms/djangoapps/instructor/management/commands/compute_grades.py

```diff
@@ -3,7 +3,7 @@
 django management command: dump grades to csv files
 for use by batch processes
 """
-from instructor.offline_gradecalc import offline_grade_calculation
+# from instructor.offline_gradecalc import offline_grade_calculation
```

Comment:
Please don't comment out code. Delete it if it isn't used anymore. Also, did you mean to leave these changes in the PR? It seems like you were using this management command for testing purposes and didn't intend to change how it works on a more permanent basis.